### DISCUSSION TITLE
favicon blue tint

### DISCUSSION THREAD

**Discussion by RuiAAPeres** [hugo-sid/hugo-blog-awesome#47] [Q&A]
I think this is my own ignorance and I apologise for this. I did the following:

1. Follow this guide [here](https://github.com/hugo-sid/hugo-blog-awesome#adding-favicon).
2. I removed the svg versions, since my icon doesn't work as an svg. 

I am getting this result on Safari:

<img width="80" alt="image" src="https://github.com/hugo-sid/hugo-blog-awesome/assets/949286/ad8d0f86-5f01-45ab-aa2f-07567821e070">

This is how my project is structured:

<img width="296" alt="image" src="https://github.com/hugo-sid/hugo-blog-awesome/assets/949286/3892a5e5-0005-48d1-9030-1ce278e5222d">

What am I missing? 

Thank you!

**Comment by hugo-sid** [comment]
Hi @RuiAAPeres.

Thanks for pointing this out this issue.

>I think this is my own ignorance and I apologise for this.

Not at all. I apologise since I made a mistake in the documentation.

## Solution

Here is the corrected version of the documentation:

After using [realfavicongenerator.net](https://realfavicongenerator.net/) to generate the favicons, you have to place the resulting files in `assets\icons` folder.

I have updated the documentation (`README.md`) to reflect this:

https://github.com/hugo-sid/hugo-blog-awesome/blob/4438214567c515feeabaa61204c4c3b1c1f0da02/README.md?plain=1#L71-L73

So, you just have to move the files present in `static\images\favicon` to `assets\icons` folder.

## Note

It's a good idea to disable cache before you check your website for the updated favicons. If you are unsure how to do this, you can visit this web page: https://nicholasbering.ca/tools/2016/10/09/devtools-disable-caching/ .

## Conclusion

Let me know if this solution worked for you. In case it did, you can mark this as the answer. Otherwise, feel free to post your observations here.

**Comment by RuiAAPeres** [comment]
I moved the icons to: 

<img width="293" alt="image" src="https://github.com/hugo-sid/hugo-blog-awesome/assets/949286/0c1fb5aa-3dd8-4d22-bb4e-f61ff558a803">

when I ran `hugo` this is the `images` folder (empty):

<img width="236" alt="image" src="https://github.com/hugo-sid/hugo-blog-awesome/assets/949286/ac379248-fc75-45dc-b1f7-7fc18be85ee6">

This is my head.html:

<img width="601" alt="image" src="https://github.com/hugo-sid/hugo-blog-awesome/assets/949286/2528ca07-e783-429e-b0ec-67f0ae7d70fb">

The icon now looks like this:

<img width="114" alt="image" src="https://github.com/hugo-sid/hugo-blog-awesome/assets/949286/0559d603-59e0-4ae4-8d8a-911a5b88fe6b">

**Comment by RuiAAPeres** [comment]
I updated the `head.html` and it's working now! Thanks

**Reply by hugo-sid** [reply]
Hi @RuiAAPeres.

I was not aware that you modified the `head.html` [partial](https://github.com/hugo-sid/hugo-blog-awesome/blob/04afe81c635bf6129c841d9c9b72b13065e4eec0/layouts/partials/head.html#L20-L25). So, I was not able to provide you with an accurate solution.

I am glad you were able to identify the solution and sort it yourself.

Feel free to ask questions relating to any other problem/issue that you may across.

Have a good day. Thanks.
